### ISSUE TITLE
Paragraphs in table cell doesn't work in Latex output

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#1574]
A table cell containing paragraphs like this:

```
+--------------+----------+-----------+-----------+
| row 1, col 1 | column 2 | column 3  | column 4  |
+--------------+----------+-----------+-----------+
| row 2        | one                              |
| row 2        |                                  |
| row 2        | two                              |
| row 2        |                                  |
| row 2        | three                            |
+--------------+----------+-----------+-----------+
| row 3        |          |           |           |
+--------------+----------+-----------+-----------+
```

Works fine in HTML output, but doesn't work in Latex output.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/1574
- Originally reported by: Peter Suter
- Originally created at: 2014-09-19T11:30:27.980

**Comment by shimizukawa** [comment]
_From Peter Suter on 2014-09-25 12:12:30+00:00_

I get this error:

```
Runaway argument?
{ one
! Paragraph ended before \multicolumn was complete.
<to be read again>
                   \par
l.231 \hline\end{tabulary}

?
! Emergency stop.
<to be read again>
                   \par
l.231 \hline\end{tabulary}

!  ==> Fatal error occurred, no output PDF file produced!
```

Slightly simpler tables work fine.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2014-09-25 13:39:53+00:00_

Thanks for reporting. I confirmed a similar error output. 

Actually, these examples don't work too.

vertical combined cell with paragraph:

```
+----------+-----------+
| one      | column 1  |
+          +-----------+
| two      | column 2  |
+----------+-----------+
```

horizontal combined cell with paragraph:

```
+----------+-----------+
| column 1 | column 2  |
+----------+-----------+
| one                  |
|                      |
| two                  |
+----------+-----------+
```

**Comment by Jellby** [comment]
The same error appears with a table like this:

```
===== ======
\     Foo
===== ======
1     2
3     4
===== ======
```

because the generated LaTeX code contains:

```
\textsf{\relax

}
```

in the first cell, which converts to a paragraph break.

**Comment by tk0miya** [comment]
I'd fixed following case at ee7e3cc.

```
===== ======
\     Foo
===== ======
1     2
3     4
===== ======
```

The other cases is hard to fix for me.
In [my repository](https://github.com/tk0miya/sphinx/tree/1574_table_cell), I tried to fix these cases with `eqparbox`.
but it still has problems below:
- got cell overflow when using `\multirow`
  
  ![cell_overflow](https://cloud.githubusercontent.com/assets/748828/11989500/1d975f04-aa42-11e5-8609-f430b97f6be7.png)
- some elements raises error (cf. `\itemize`)

If anyone knows good solution in LaTeX, please let me know.
I'll incorporate it into Sphinx.

**Comment by xuhdev** [comment]
@tk0miya  The following LaTeX code works for me for multicolumn. It seems that longtable supports paragraphs in multicolumn out of box, but I don't know why tabulary is not working. Also see [here](https://tex.stackexchange.com/questions/39976/is-it-possible-to-have-multiple-paragraphs-inside-a-multicolumn-using-xcolor-or) (the question itself, not the answer) for another example. For multirow, maybe [makecell](https://www.ctan.org/pkg/makecell?lang=en) can help (see the manual section 4: you can manually add linebreaks in `\multirowcell` to simulate paragraphs).

``` latex
\documentclass{article}

\usepackage{longtable}

\begin{document}

\begin{longtable}{|c|c|}
\hline
column 1 & column 2 \\
\hline \multicolumn{2}{|p{15cm}|}{
one

two
}\\
\hline\end{longtable}

\end{document}
```

**Comment by jfbu** [comment]
@Jellby 

I tested your input

```
===== ======
\     Foo
===== ======
1     2
3     4
===== ======
```

on current master branch (soon Sphinx 1.5 beta) and the PDF builds ok.

**Comment by jfbu** [comment]
ah, sorry @tk0miya already said he had fixed it at ee7e3cc, sorry.
